test/gen.sh

```diff
@@ -28,13 +27,19 @@ mkdir -p "$PROTOCONF_OUT" "$LOADER_OUT" "$CHECKER_OUT"
 
 # generate protoconf files
 ${PROTOC} \
---go_out="$PROTOCONF_OUT" \
---go_opt=paths=source_relative \
---go-tableau-loader_out="$LOADER_OUT" \
---go-tableau-loader_opt=paths=source_relative \
---go-tableau-checker_out="$CHECKER_OUT" \
---go-tableau-checker_opt=paths=source_relative,out="$CHECKER_OUT" \
---proto_path="$PROTOBUF_PROTO" \
---proto_path="$TABLEAU_PROTO" \
---proto_path="$PROTOCONF_IN" \
-"$PROTOCONF_IN"/*
\ No newline at end of file
+    --go_out="$PROTOCONF_OUT" \
+    --go_opt=paths=source_relative \
+    --go-tableau-loader_out="$LOADER_OUT" \
+    --go-tableau-loader_opt=paths=source_relative \
+    --proto_path="$PROTOBUF_PROTO" \
+    --proto_path="$TABLEAU_PROTO" \
+    --proto_path="$PROTOCONF_IN" \
+    "$PROTOCONF_IN"/*
+
+${PROTOC} \
+    --go-tableau-checker_out="$CHECKER_OUT" \
+    --go-tableau-checker_opt=paths=source_relative,out="$CHECKER_OUT" \
+    --proto_path="$PROTOBUF_PROTO" \
+    --proto_path="$TABLEAU_PROTO" \
+    --proto_path="$PROTOCONF_IN" \
+    "$PROTOCONF_IN"/test_conf.proto
```

Comment:
Add comment: Intended for testing: DO NOT generate "*.check.go" for item_conf.proto